simd/src/Kokkos_SIMD.hpp

```diff
@@ -62,7 +62,8 @@
 #include <Kokkos_SIMD_AVX512.hpp>
 #endif
 
-#if defined(KOKKOS_ARCH_ARM_NEON)
+// FIXME: If and when NVCC supports NEON intrinsics reenable
```

Comment:
```suggestion
// FIXME_NVCC: If and when NVCC supports NEON intrinsics reenable
```

Reply:
Are you OK with not putting the NVCC conditional here and just relying on CMake? If so, I can also remove this comment.